### PR TITLE
Am/hotfix idl changes

### PR DESCRIPTION
**Pull request by Arjunica**
change of places' IDL file

### CODE REVIEW

**Comment by QuinnWass** [comment]
I will review. I would like @galaxyonfire to approve too.

File: mergedFiles/places_client.c [hunk 1]
```diff
@@ -12,8 +12,11 @@ places_server_1(char *host)
 {
 	CLIENT *clnt;
 	places_ret  *result_1;
-	char  find_closest_airport_given_name_1_arg;
-
+	places_param_request  find_closest_airport_given_name_1_arg;
```
Comment (QuinnWass) [inline mergedFiles/places_client.c:15]: This is temporary and was required for the marshalling.

File: mergedFiles/places_client.c [hunk 2]
```diff
@@ -27,6 +30,8 @@ places_server_1(char *host)
 		clnt_perror (clnt, "call failed");
 	}
 
+	printf("%i", result_1->status);
```
Comment (QuinnWass) [inline mergedFiles/places_client.c:33]: This was to confirm pass through

File: mergedFiles/places_server.c [hunk 3]
```diff
@@ -8,12 +8,12 @@
...
 
 places_ret *
-find_closest_airport_given_name_1_svc(char *argp, struct svc_req *rqstp)
+find_closest_airport_given_name_1_svc(places_param_request *argp, struct svc_req *rqstp)
 {
 	static places_ret  result;
 	result.status = -1;
 
-	char host[10] = "localhost\0";
+	char host[11] = "localhost\0";
```
Comment (QuinnWass) [inline mergedFiles/places_server.c:16]: This can be changed later

**Review by QuinnWass — APPROVED** [review]
Sweet! Can confirm it compiles and runs!

**Review by galaxyonfire — APPROVED** [review]
look good 2 me

**Pull request merged**